Dockerfile

```diff
@@ -442,6 +442,31 @@ ENV PATH="/usr/libexec/rust/bin:$PATH" LD_LIBRARY_PATH="/usr/libexec/rust/lib"
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-libc as sdk-bootconfig
+
+USER root
+
+ARG KERNELVER="5.10.102"
+
+RUN \
+  mkdir -p /usr/libexec/tools && \
+  chown -R builder:builder /usr/libexec/tools
+
+USER builder
+WORKDIR /home/builder
+COPY ./hashes/kernel /home/builder/hashes
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  curl --fail https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-${KERNELVER}.tar.xz -o linux-${KERNELVER}.tar.xz && \
+  tar -xf linux-${KERNELVER}.tar.xz && rm linux-${KERNELVER}.tar.xz
+
+WORKDIR /home/builder/linux-${KERNELVER}
+RUN \
+  make -C tools/bootconfig && \
```

Comment:
You might be able to do something like `bindir=/usr/libexec/tools/bootconfig` and just install it there rather than copying individual files.